<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaCredenciales</class>
 <widget class="QWidget" name="ventanaCredenciales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inicio de sesion</string>
  </property>
  <widget class="QPushButton" name="buttonAceptar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancelar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNombre">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineContrasenia">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Contrase√±a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRegistro">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrarse</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
